packages/mjml-button/src/index.js

```diff
@@ -1,141 +1,190 @@
-import { MJMLElement, helpers } from 'mjml-core'
-import React, { Component } from 'react'
+import {MJMLElement, helpers} from 'mjml-core'
+import React, {Component} from 'react'
 
-const tagName = 'mj-button'
-const parentTag = ['mj-column', 'mj-hero-content']
-const endingTag = true
+const tagName               = 'mj-button';
```

Comment:
Well, I don't share this opinion. Some case still require `;` but it's doesn't add much to add semicolons, this will remain a debate in the community